python/paddle/tensor/manipulation.py

```diff
@@ -737,6 +737,15 @@ def shard_index(
             >>> print(shard_label.numpy())
             [[-1]
              [ 1]]
+
+        A two-dimensional tensor with a shape of [2, 1], through the shard_index operation, when index_num = 20, nshards = 2, and shard_id = 0, calculates
+        the shard size shard_size = (20 + 2 - 1) // 2 = 10. For the element values in label, such as 16, because 16 >= 10 (not within the [0, 10) interval
+        of shard 0), its value is set to ignore_value (i.e., -1) according to the formula; while the element value 1, since 0 <= 1 < 10 (within shard 0),
+        the updated value is 1 - 0 * 10 = 1, and finally a new tensor with the same shape [2, 1] but some elements changed (such as shard_label) is obtained.
```

Comment:
```suggestion
        As shown below, a 2D tensor with shape [2, 1] is updated using the ``shard_index`` operation. With ``index_num=20``, ``nshards=2``, and ``shard_id=0``, the shard size is calculated as ``shard_size = (20 + 2 - 1) // 2 = 10``. 
        For each element in label: If the value is within the shard range ``[0, 10)``, it is adjusted to its offset in the range. For example, ``1`` is updated to ``1 - 0 * 10 = 1``; Otherwise, it is set to the ``ignore_value`` (default ``-1``). For example, ``16`` is outside the range ``[0, 10)`` and becomes ``-1``.
```